pycomlink/processing/nearby_rain_retrival.py

```diff
@@ -198,6 +198,4 @@ def nearby_rainfall_retrival(
     if F_value_correction is True:
         R = R.where(~(F <= -32.5))
```

Comment:
Should this threshold that is used here be a kwarg with default = -32.5?

Reply:
`F_value_treshold` is a new kwarg now replacing `F_value_correction`.